modules/ROOT/pages/dataweave-pattern-matching.adoc

```diff
@@ -97,9 +97,12 @@ output application/json
 }
 ----
 
-For use cases where you need a boolean result based on whether a value
-matches or not, you can use the `matches` function instead of using pattern
-matching.
+==Other Ways to Pattern Match a String
+For use cases where the left-side input argument is a String, you can also use the dw::Core module's `match` function to test the string against a regex pattern. 
+
+For use cases where you need a boolean result based on whether a String left-side input argument matches some regex pattern, you can use the `matches` function instead of using pattern matching or the dw::Core module's `match` function.
```

Comment:
```suggestion
For use cases that require a Boolean result based on whether a String left-side input argument matches a regex pattern, you can use the `matches` function instead of using pattern matching or the dw::Core module's `match` function.
```